### PR TITLE
feat: hide trash button during collaboration

### PR DESCRIPTION
**Pull request by ad1992**
https://user-images.githubusercontent.com/11256141/163429110-4ca8633d-7228-45cd-b80c-51963b6158d1.mp4

### CODE REVIEW

**Comment by dwelle** [comment]
I'd only hide it visually so there's no jump. So using a combo of `visibility: hidden` & `pointer-events: none`, or replacing it with a placeholder that has the same width.

File: .env.development [hunk 1]
```diff
@@ -7,6 +7,6 @@ REACT_APP_LIBRARY_BACKEND=https://us-central1-excalidraw-room-persistence.cloudf
 REACT_APP_PORTAL_URL=http://localhost:3002
 # Fill to set socket server URL used for collaboration.
 # Meant for forks only: excalidraw.com uses custom REACT_APP_PORTAL_URL flow
-REACT_APP_WS_SERVER_URL=
+REACT_APP_WS_SERVER_URL=http://localhost:3002
```
Comment (dwelle) [inline .env.development:10]: this actually we shouldn't commit :) — put into `.env.local`

Comment (ad1992) [reply .env.development:10]: we aren't storing any user secrets in this file so should be fine to commit? That way Devs will not need to update it as well.

File: public/index.html [hunk 2]
```diff
@@ -161,7 +161,6 @@
           -ms-user-select: auto;
           user-select: auto;
         }
-      }
```
Comment (dwelle) [inline public/index.html:164]: what's this?

File: .env.development (hunk 1 above)
Comment (dwelle) [reply .env.development:8]: ok, sounds reasonable. Changed the doc to reflect this

File: public/index.html (hunk 2 above)
Comment (ad1992) [reply public/index.html]: Was a typo in prod due to which the media rules were never getting applied, fixed now

**Review by dwelle — APPROVED** [review]

**Pull request merged**